hathor/p2p/resources/status.py

```diff
@@ -170,7 +184,17 @@ def render_GET(self, request):
                                         },
                                         'dag': {
                                             'first_timestamp': 1539271481,
-                                            'latest_timestamp': 1539271483
+                                            'latest_timestamp': 1539271483,
+                                            'best_block_tips': [
+                                                {
+                                                    'hash': '000006c',
+                                                    'height': 0
+                                                }
+                                            ],
+                                            'best_block': {
+                                                'hash': '000006c',
```

Comment:
I'd put the full hash of the genesis block, just to make it really clear how it works, that is, it is not the first few characters of the hash.

Reply:
Done at https://github.com/HathorNetwork/hathor-core/pull/549/commits/fc94a87f00db8fc15384ea407db3b1dd384d70dc